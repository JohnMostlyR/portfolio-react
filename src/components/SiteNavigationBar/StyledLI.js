import styled from 'styled-components';

import mq from '../../styles/templates/mediaQueries';

const StyledLI = styled.li`
  vertical-align: middle;

  ${mq.m`margin-top: 2vw;`}

  &:not(:first-child) {
    ${mq.m`margin-left: 1vh;`}
  }
`;

export default StyledLI;
