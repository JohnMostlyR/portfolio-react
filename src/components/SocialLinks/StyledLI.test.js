import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import StyledLI from './StyledUL';

describe('StyledLI', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<StyledLI/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});