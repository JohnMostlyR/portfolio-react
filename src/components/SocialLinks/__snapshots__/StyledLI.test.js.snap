// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledLI Should render and match the snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
}

<ul
  className="c0"
/>
`;
